src/artemis/experiment_plans/optimise_attenuation_plan.py

```diff
@@ -58,6 +79,87 @@ def arm_devices(xspress3mini, zebra):
     yield from arm_zebra(zebra)
 
 
+def deadtime_is_transmission_optimised(
```

Comment:
Could: It might be cleaner if the function returned whether you're optimised and then which direction to move in, rather than the flip

Reply:
Could: It would be even more clear if we just separate it out into two functions, one to work out if we're there and one to work out if to flip. It might lead to some repeated work but clarity is more important than efficiency here

Reply:
Should: Can we have some separate tests on this function